themes/classic/_dev/package.json

```diff
@@ -22,7 +22,7 @@
     "flexibility": "^1.0.5",
     "jquery": "^2.1.4",
     "material-design-icons": "^2.1.3",
-    "node-sass": "^4.5.0",
+    "node-sass": "^4.11.0",
```

Comment:
Reason to update node-sass?

Reply:
Because it forces to me update it to work

Reply:
But you don't update the package-lock.json file :thinking:

Reply:
I added it

Reply:
What was the error message? What version of node are you using?

Reply:
I'm using 6.9.0 version

Reply:
You're using a very old version of Node, try installing 8.x

You can use NVM for that: https://github.com/creationix/nvm

Reply:
You must upgrade your node version since the LTS is 10 and current is 11. We're using at least node 8.

Reply:
You can also use: https://github.com/riywo/ndenv

Reply:
I made a mistake. I'm 11.8.0 I looked for npm when I said 6.9.0

Reply:
That's why :P

Reply:
We can keep this commit.